templates/common/_base/units/machine-config-daemon-host.service

```diff
@@ -3,6 +3,7 @@ enabled: true
 contents: |
   [Unit]
   Description=Machine Config Daemon Initial
+  EnvironmentFile=/etc/machine-config-daemon/proxy
```

Comment:
What if we just inject them everywhere?
https://github.com/coreos/bugs/issues/1507#issuecomment-240233869

This way the env vars would be picked up automatically by e.g. a user logging in via `ssh` as well as e.g. `cri-o.service`.  But I don't think they'd be picked up by `oc debug node/`...whole other can of worms there.

Reply:
Coming from Slack conversations, this enablement is ad-hoc for our services that use external services needing a proxy

Reply:
The scope of the MCO is everything inclusive of kernel to kubelet right?  The way you say "our services" seems to argue for just handling one of our core functions, but I would say we do want the same thing to work if a user runs `podman` directly at least.

Actually it's not clear to me...the MCO is also going to take care of this for cri-o/kubelet right?